docs/template_plugin/tests/functional/shared_tests_instances/single_layer_tests/convolution.cpp

```diff
@@ -78,6 +80,8 @@ INSTANTIATE_TEST_SUITE_P(Convolution2D_AutoPadValid, ConvolutionLayerTest,
                                 ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
                                 ::testing::Values(InferenceEngine::Layout::ANY),
                                 ::testing::Values(InferenceEngine::Layout::ANY),
+                                ::testing::Values(std::vector<std::vector<size_t >>({}),
+                                                  std::vector<std::vector<size_t >>({{1, 10}, {3, 30}, {30, 300}, {30, 300}})),
```

Comment:
We also need some global constant for fully undefined range smth like `std::vector<std::pair<size_t, size_t>> UNDEFINED_RANGE = { -1, -1 };`. It will make undefined range initialization more convenient than placing  `{ -1, -1 }` everywhere we need undefined bounds.

Reply:
#define UNDEFINED_RANGE and NULL_RANGE seem to be right for your comment. If you can give another cases, I'll try them. Thanks for your comment.